styleguide.config.js

```diff
@@ -6,6 +6,9 @@ const propFilter = (prop) => {
   if (prop.description.length === 0) {
     return false;
   }
+  // if (['margin', 'mx', 'my', 'mt', 'mr', 'mb', 'ml'].includes(prop.name)) {
```

Comment:
Is this needed for something?

Reply:
It will hide the margin props from props list in the next PR

Reply:
Left here commented out so it was easier to see/test which components had the margin props support added